browser/package.json

```diff
@@ -12,6 +12,19 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "parcel": "^2.6.2"
+    "@babel/preset-env": "^7.18.6",
+    "@babel/preset-react": "^7.18.6",
+    "babel-jest": "^28.1.2",
+    "jest": "^28.1.2",
+    "parcel": "^2.6.2",
+    "process": "^0.11.10",
```

Comment:
What's process being used for?

Reply:
I don't think we should be using this unless we have a very good reason not to be using the built-in process library from node.  Is this a peer dependency or something?

Reply:
I'm actually not sure where this came from. I looked up how to install React without create-react-app, but looking at my terminal history, I didn't explicitly install process, so maybe it's a dependency from something else?

Reply:
I don't think it'd have ended up in the package.json if that were the case -- mind deleting it and reinstalling packages to generate a new lock file?

Reply:
Can we remove this still?

Reply:
Weird, I thought I did earlier on. Will delete `process` again